jupyter_archive/handlers.py

```diff
@@ -213,7 +213,7 @@ def on_connection_close(self):
         self.flush_cb.stop()
 
 
-class ExtractArchiveHandler(JupyterHandler):
+class ExtractArchiveHandler(APIHandler):
```

Comment:
To return json response, we need to use `APIHandler` instead of `JupyterHandler`.
Do you have any concern to use it?

Reply:
Thank you for your feedback.
It seems that `APIHandler.write_error()` is necessary and sufficient, so I ported the process as it was.
If it contains unnecessary processing or we need a simpler implementation, please let us know.